shared/macros-bash.jinja

```diff
@@ -1026,6 +1026,7 @@ if ! grep -qE '^\s*auth\s+required\s+pam_faillock\.so\s+(preauth silent|authfail
     sed -i --follow-symlinks '/^auth.*sufficient.*pam_unix.so.*/a auth        required      pam_faillock.so authfail' "{{{ pam_file }}}"
     sed -i --follow-symlinks '/^account.*required.*pam_unix.so.*/i account     required      pam_faillock.so' "{{{ pam_file }}}"
 fi
+sed -Ei 's/(auth.*)(\[default=die\])(.*pam_faillock.so)/\1required     \3/g'
```

Comment:
missing file path of the file to be modified

Reply:
Thanks for catching this.